torch/linalg/__init__.py

```diff
@@ -8,6 +8,51 @@
 # Note: This not only adds doc strings for functions in the linalg namespace, but
 # also connects the torch.linalg Python namespace to the torch._C._linalg builtins.
 
+cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
+linalg.cholesky(input) -> Tensor
+
+Returns the Cholesky decomposition.
+
+Computes the Cholesky decomposition of a Hermitian (or symmetric for real-valued matrices)
+positive-definite matrix :math:`A` or for batches of Hermitian positive-definite matrices.
+The returned matrix ``L`` is lower-triangular, and
+the decomposition has the form:
+
+.. math::
+
+    A = LL^H
+
+If :attr:`input` is a batch of Hermitian positive-definite
+matrices, then the returned tensor will be composed of lower-triangular Cholesky factors
+of each of the individual matrices.
+
+.. note:: If the :attr:`input` is not Hermitian positive-definite matrix a RuntimeError is raised
+          saying that the input is singular and mentioning which minor of the input matrix is not positive-definite.
+
+.. note::
```

Comment:
Don't make this a note, it can go write after the preamble (or after a separate mathematical expression if the doc has one). "Supports real inputs and, on the CPU only, complex inputs."